docs/content/docs/dev/configuration/overview.md

```diff
@@ -181,22 +171,6 @@ rootProject.name = 'quickstart'
 bash -c "$(curl https://flink.apache.org/q/gradle-quickstart.sh)" -- {{< version >}} {{< scala_version >}}
 ```
 {{< /tab >}}
-{{< tab "sbt" >}}
-You can scaffold a new Flink project with the following [giter8 template](https://github.com/tillrohrmann/flink-project.g8)
-and the `sbt new` command (which creates new build definitions from a template) or use the provided quickstart bash script.
-
-### sbt template
-
-```bash
-$ sbt new tillrohrmann/flink-project.g8
```

Comment:
Why was this removed?

Reply:
We talked to @tillrohrmann about it - The SBT example was still targeting 0.13.13 while SBT 1.6.1 is the latest version. We decided to first remove it and create a follow-up ticket for adding an SBT example later.

Reply:
I had already removed the standalone page on SBT and had forgotten to remove this section.